### PR TITLE
Add playready support

### PR DESCRIPTION
**Pull request by arunshan**
### This PR will...
Adds support for playready DRM.
There is already a PR https://github.com/Tubitv/hls.js/pull/3 against master but we wanted the integration against a feature branch.

### Why is this Pull Request needed?
It adds support for playready which is used on Edge browsers and devices that have edge like browsers.

### Are there any points in the code the reviewer needs to double check?
Please verify if the changes are exactly the same as https://github.com/Tubitv/hls.js/pull/3/files

### Resolves issues:
I did not adds any tests. I do not know what is the process there. Do we need to add tests or are we doing that later?

### CODE REVIEW

File: src/controller/eme-controller.js [hunk 1]
```diff
@@ -63,6 +63,26 @@ const createWidevineMediaKeySystemConfigurations = function (audioCodecs, videoC
...
+    // distinctiveIdentifier: "not-allowed", // or "required" ?
+    // sessionTypes: ['temporary'],
+    audioCapabilities: [
+      // { contentType: 'audio/mp4; codecs="mp4a.40.2"' }
+    ],
+    videoCapabilities: [
+      // { contentType: 'video/mp4; codecs="avc1.42E01E"' }
+    ]
+  };
+  // TODO add proper video and audio codecs for playready
```
Comment (springuper) [inline src/controller/eme-controller.js:80]: should we do it right now?

**Review by springuper — APPROVED** [review]
LGTM.

Thanks!

**Comment by springuper** [comment]
@arunshan I'm going to merge it first. Thank you for your contribution!

File: src/controller/eme-controller.js (hunk 1 above)
Comment (arunshan) [reply src/controller/eme-controller.js:80]: Sorry didnt see this. We can do this later.

**Pull request merged**